bridges/MastodonBridge.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+class MastodonBridge extends FeedExpander {
+
+	const MAINTAINER = 'husim0';
+	const NAME = 'Mastodon Bridge';
+	const CACHE_TIMEOUT = 900; // 15mn
+	const DESCRIPTION = 'Returns toots';
+	const URI = 'https://mastodon.social';
+
+	const PARAMETERS = [[
+		'canusername' => [
+			'name' => 'Canonical username (ex : @sebsauvage@framapiaf.org)',
+			'required' => true,
+		],
+		'norep' => [
+			'name' => 'Without replies',
+			'type' => 'checkbox',
+			'title' => 'Only return initial toots'
+		],
+		'noboost' => [
+			'name' => 'Without boosts',
+			'required' => false,
+			'type' => 'checkbox',
+			'title' => 'Hide boosts'
+		]
+	]];
+
+	public function getName(){
+		switch($this->queriedContext) {
+		case 'By username':
+			$param = 'canusername';
```

Comment:
This function can be simplified if you return immediately.

```suggestion
			return $this->getInput('canusername');
```

In this case the `break;` and the last line `return $this->getInput($param);` can be removed entirely.